### ISSUE TITLE
Feature: Simple Auth page

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#608]
A simple HTML page that is easily put into an iframe and allows scanning of the QR code, maybe shows some logs and session state.

**Comment by DeLuca92** [comment]
Maybe, total.js is a good way. 
To make a auth page and show logs.

**Comment by smashah** [comment]
thanks for the suggestion deLuca. Will be working on the majority of these issues over the weekend. v2.0.0 of open-wa will be amazing!

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.97](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.96...1.9.97) (2020-07-09)

- build(deps): bump puppeteer-extra from 3.1.9 to 3.1.12 [`#633`](https://github.com/open-wa/wa-automate-nodejs/pull/633)
- build(deps): bump puppeteer-extra-plugin-devtools from 2.2.8 to 2.2.11 [`#632`](https://github.com/open-wa/wa-automate-nodejs/pull/632)
- build(deps): bump puppeteer-extra-plugin-stealth from 2.4.9 to 2.4.13 [`#635`](https://github.com/open-wa/wa-automate-nodejs/pull/635)
- build(deps-dev): bump @types/node from 14.0.19 to 14.0.20 [`#634`](https://github.com/open-wa/wa-automate-nodejs/pull/634)
- build(deps-dev): bump @types/node from 14.0.18 to 14.0.19 [`#631`](https://github.com/open-wa/wa-automate-nodejs/pull/631)
- build(deps): bump chrome-launcher from 0.13.3 to 0.13.4 [`#630`](https://github.com/open-wa/wa-automate-nodejs/pull/630)
- build(deps-dev): bump @types/node from 14.0.14 to 14.0.18 [`#627`](https://github.com/open-wa/wa-automate-nodejs/pull/627)
- build(deps-dev): bump @types/express from 4.17.6 to 4.17.7 [`#628`](https://github.com/open-wa/wa-automate-nodejs/pull/628)
- build(deps-dev): bump auto-changelog from 2.1.0 to 2.2.0 [`#624`](https://github.com/open-wa/wa-automate-nodejs/pull/624)
- Release 1.9.97 [`cb6e8b8`](https://github.com/open-wa/wa-automate-nodejs/commit/cb6e8b82f66f7d791b2b2559aca78d693d8b52f3)
- Feature: Implemented pop up auth window #608 [`108ac2a`](https://github.com/open-wa/wa-automate-nodejs/commit/108ac2a2ee85cffd15aed1a4b38c05e016c4f73e)
- Better init logging [`c5bb467`](https://github.com/open-wa/wa-automate-nodejs/commit/c5bb46741446bfed307abc2c5f2564cd6b8eee5a)
- optimize launch logo [`7c6ebd4`](https://github.com/open-wa/wa-automate-nodejs/commit/7c6ebd4ea2ff2b99e68831f01c0ab57baf99f939)
- Feature: Implemented `skipSessionSave` #606 [`58280c2`](https://github.com/open-wa/wa-automate-nodejs/commit/58280c23b9d9a2bab3caa5202f377c05d040a3a0)
- onStateChanged docs fix [`bdd7c75`](https://github.com/open-wa/wa-automate-nodejs/commit/bdd7c759f5b07d948d9928e13452334e55504d6c)
- Better error messages for sendMessage [`29b9488`](https://github.com/open-wa/wa-automate-nodejs/commit/29b94888010797473ecff9c6332d6e921c4db0a3)
- update supported WA version [`1cc0c69`](https://github.com/open-wa/wa-automate-nodejs/commit/1cc0c6957d74e0ec323761ba32eb6db89d686b39)
- optimise business profie getter [`3e66d69`](https://github.com/open-wa/wa-automate-nodejs/commit/3e66d6920b5d18c5bfee006bc1a752172e308b41)

**Comment by smashah** [comment]
In v1.9.97 you will now be able to use a pop up to authenticate the session or just check the start up logs.

https://open-wa.github.io/wa-automate-nodejs/interfaces/configobject.html#popup

```javascript
...

//Open browser to see logs and QR code. Usually it will open on port 3000
create({
	popup: true
})

//or you can set your own preferred port. If the preferred port is busy, it will automatically find an open port
create({
	popup: 1234
})

```
